internal/logger/config.go

```diff
@@ -0,0 +1,27 @@
+package logger
+
+import (
+	"os"
+	"time"
+
+	"github.com/dicedb/dice/config"
+	"github.com/rs/zerolog"
+)
+
+var l zerolog.Logger
+
+type CtxKey string
+
+// Context keys used across the project
+const (
+	RequestID CtxKey = "request_id"
```

Comment:
Can we add them to the context.Context instead of logs? The logs can use the values from context when needed. Addditionally, other resources in the code can use them as well.

Reply:
Actually this is just a constant variable, imo we need another package to keep constants. If you see in the PR I'm demonstrating usage of these variables to keep track of shard.